test/database/create_storage_tables_user_attachments_migration.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class CreateStorageTablesUserAttachmentsMigration < ActiveRecord::Migration[7.0]
+  def change
+    create_table :storage_tables_user_attachments, force: true do |t|
+      t.string :name,         null: false
+      t.string     :filename, null: false
```

Comment:
De filename is een variable width column. Door deze als laatste column in de table te plaatsen wordt de table kleiner. Dus eerst alle columns met een fixed width en daarna de variable width columns.